src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -1668,12 +1699,69 @@ else if (v instanceof ListValue)
             }
         });
 
+        //setter of ironman flower, blueness of skull, carryingitem 
+        /*
+        put("carrying_item",(e,v)->{
+                if (e instanceof ServerPlayer pl){
+                    ItemStack item=getItemStackFromValue(v);
+                    item=item.copy();
+                    pl.containerMenu.setCarried(item);
+                    pl.containerMenu.broadcastFullState();
+                }
+        });
+        */
+        put("skull_dangerous",(e,v)->{
+            if (e instanceof WitherSkull w){
+                w.setDangerous(v.getBoolean());
+            }
+            
+        });
+        put("offer_flower",(e,v)->{
+            if (e instanceof IronGolem ig){
+                ig.offerFlower(v.getBoolean());
+            }
+            
+        });
+        put("item", (e, v) -> {
+                ItemStack item=getItemStackFromValue(v).copy();
+                if(e instanceof ItemEntity iteme)            
+                    iteme.setItem(item);
+                if(e instanceof ItemFrame iteme)
+                    iteme.setItem(item);  
+        });
```

Comment:
I can't comment below cos its not modified, but you can remove the checkbox for item (line 1733)